drivers/hud_apr/app/models/hud_apr.rb

```diff
@@ -7,6 +7,8 @@
 # frozen_string_literal: true
 
 module HudApr
+  TodoOrDie('Set APR Default Generator on Staging to FY 2026 version', by: '2025-09-01')
```

Comment:
This is going to come due next week, do we maybe want to just set it now?  Or better yet, can we just rely on `hud_csv_version` to determine if we should be using FY2024 or FY2026?